### PR TITLE
Massive star alias updates

### PR DESCRIPTION
**Pull request by Rplesha**
Capitalization bug fix and adding in one more alias. Also removing the "NGC 3109" from the simbad name because that is handled for display in the target metadata db file.

### CODE REVIEW

**Review by jduval82 — COMMENTED** [review]
Changes to alias file and exposure list look good. Not sure about changes to the calibration DB, but other reviewers know more about this

File: utils/data/calibration_metadata/ullyses_calibration_db.csv [hunk 1]
```diff
@@ -4707,10 +4717,10 @@ leqj2frsq,HST,COS,16596,FUV,G160M,N/A,1623,PSA,4,59662.02560655,2022-03-24T00:36
 oeqj2s010,HST,STIS,16596,NUV-MAMA,G230L,Clear,2376,52X2,N/A,59661.87277669,2022-03-23T20:56:47.906,V-SY-CHA,V-SY-CHA,True,False,False,False,STIS,Cha I,True,False,
 oeqj2s020,HST,STIS,16596,CCD,G430L,Clear,4300,52X2,N/A,59661.89057743,2022-03-23T21:22:25.890,V-SY-CHA,V-SY-CHA,True,False,False,False,STIS,Cha I,True,False,
 oeqj2s030,HST,STIS,16596,CCD,G750L,Clear,7751,52X2,N/A,59661.89667706,2022-03-23T21:31:12.898,V-SY-CHA,V-SY-CHA,True,False,False,False,STIS,Cha I,True,False,
-oclv02010,HST,STIS,13775,CCD,G750L,Clear,7751,52X2,N/A,57096.10141035,2015-03-15T02:26:01.854,V-SZ-CHA,V-SZ-CHA,True,False,False,False,STIS,,False,True,
```
[the remark this answers is not in the document]
Comment (Rplesha) [reply utils/data/calibration_metadata/ullyses_calibration_db.csv:4710]: Never mind. The region was added, not removed. It is correct now.

**Review by jotaylor — COMMENTED** [review]
Everything looks good. Some of the cal DB diffs seem to just be from rows moving around within the CSV, is that correct? (Others actually changed).

**Comment by Rplesha** [comment]
@jotaylor yes--all of the datasets are still there but are rearranged. I'm not sure why they are rearranged, but I don't think it's too important to hunt down unless you disagree.

**Pull request merged**